packages/mineral-ui-tokens/scripts/formats.js

```diff
@@ -59,22 +82,40 @@ const formatProperties = ({
       return acc;
     }, {});
 
+const namedExports = (tokens, exportFormat) =>
+  Object.entries(tokens)
+    .map(exportFormat)
+    .join('\n');
+
+const rampGroupFilters = {
+  black: 'originalValue',
+  danger: 'name',
+  gray: 'originalValue',
+  success: 'name',
+  theme: 'name',
+  warning: 'name',
+  white: 'originalValue'
+};
+
+const removeTheoValueQuotes = (value) =>
+  typeof value === 'string' ? value.replace(/'/g, '') : value;
+
 module.exports = {
   categorizedJsExports: (input) => {
-    return defaultExport(prettifyJson(formatCategories(input)));
+    return defaultExport(formatOutput(formatCategories(input)));
```

Comment:
Great improvement overall in this file! I think formatOutput makes a lot more sense & is universal.